src/MiniCover.HitServices/TestMethodUtils.cs

```diff
@@ -31,10 +44,15 @@ private static bool HasPdb(MethodBase methodBase)
             lock (Lock)
             {
                 if (AssemblyHasPdbCache.TryGetValue(location, out var hasPdb)) return hasPdb;
-                hasPdb = File.Exists(Path.ChangeExtension(location, ".pdb"));
+                hasPdb = IsNotATestFrameworkAssembly(methodBase.DeclaringType.Assembly) && File.Exists(Path.ChangeExtension(location, ".pdb"));
                 AssemblyHasPdbCache.Add(location, hasPdb);
                 return hasPdb;
             }
         }
+
+        private static bool IsNotATestFrameworkAssembly(Assembly assembly)
```

Comment:
I slightly prefer a positive method: IsATestFrameworkAssembly
And negate where we call the method.
But this is not a blocker.